packages/pangea-sdk/pangea/request.py

```diff
@@ -5,20 +5,22 @@
 import json
 import logging
 import time
-from typing import Dict, List, Optional, Tuple, Type, Union
+from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, Type, Union
 
-import aiohttp
 import requests
 from requests.adapters import HTTPAdapter, Retry
-from requests.structures import CaseInsensitiveDict
 from requests_toolbelt import MultipartDecoder  # type: ignore
+from typing_extensions import TypeVar
 
 import pangea
 import pangea.exceptions as pe
 from pangea.config import PangeaConfig
 from pangea.response import AttachedFile, PangeaResponse, PangeaResponseResult, ResponseStatus, TransferMethod
 from pangea.utils import default_encoder
 
+if TYPE_CHECKING:
+    import aiohttp
```

Comment:
If you do not import this package won't it fail in all the lines it's called? Shouldn't you add this check (`if TYPE_CHECKING` ) in all those lines too?

Reply:
That not necessary, no.